northd/ovn_northd.dl

```diff
@@ -1306,16 +1306,28 @@ for (&Switch(.dpname = dpname)) {
          .external_ids     = map_empty())
 }
 
+relation HasEventElbMeter(has_meter: bool)
+
+HasEventElbMeter(not set_is_empty(result)) :-
```

Comment:
@LorenzoBianconi, I've been thinking more about this and i think this will create problems if there's no `nb.Meter` record. Then the join between `RouterLBVIP` and `HasEventElbMeter` will return 0 records.

I think this should be:
```
relation HasEventElbMeter(has_meter: bool)

HasEventElbMeter(true) :-
    nb.Meter(.name = "event-elb").

HasEventElbMeter(false) :-
    nb.NB_Global(._uuid = uuid),
    not nb.Meter(.name = "event-elb").
```

@ryzhyk What do you think? Is there a better way to do this in DDlog?

Reply:
Sorry, I only now realized this PR is DDlog-related. Will study it now.

Reply:
@dceara's suggestion is correct. The semantics of aggregation can be confusing in that aggregating over an empty set of records produces no output records as opposed to a single output record with an empty set.

We should consider adding a form of aggregation operator that implements the other semantics.

@dceara's code is indeed how you do this in DDlog at the moment.

Reply:
@ryzhyk to be fair the first version was also because of my incorrect suggestion to @LorenzoBianconi offline :)